### PR TITLE
[hi] Localize /en/docs/reference/glossary/istio.md

### PR DESCRIPTION
**Pull request by sachinparihar**
Fixes: #37514

### CODE REVIEW

**Comment by chiukapoor** [comment]
Hello @sachinparihar, It seems like your tests are failing as characters `[]` are not allowed to be used in the commit message. To fix the issue you may squash all your commits & retry with a fresh commit message.

Please correct me if I am wrong @bishal7679

**Comment by Garima-Negi** [comment]
> Details

There are other PRs like https://github.com/kubernetes/website/pull/37872 that include `[]` in the PR title and tests still pass.

**Review by Garima-Negi — APPROVED** [review]
Content LGTM! 
Will need to fix the tests to merge

**Review by bishal7679 — APPROVED** [review]
Looks good now!
/lgtm

**Review by Babapool — COMMENTED** [review]
Hello @sachinparihar , I just wanted to confirm whether you are intending to localize the `istio.md` file into Hindi. If so then can you please check if you localizing the correct file, because while reviewing I am seeing that the changes are being done another file (in this case the `kubectl.md`). Thanks.

**Comment by sachinparihar** [comment]
I think everything is fine now.

File: content/hi/docs/reference/glossary/istio.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+aka: 
+tags:
+- networking
+- architecture
+- extension
+---
+ एक खुला प्लैटफ़ॉर्म (कुबेरनेट्स-विशिष्ट नहीं) जो माइक्रोसर्विसेज को एकीकृत करने, यातायात प्रवाह को प्रबंधित करने, नीतियों को लागू करने और कुल टेलीमेट्री डेटा को एकीकृत करने का एक समान तरीका प्रदान करता है।
+<!--more--> 
+
+इस्तियो को जोड़ने के लिए एप्लिकेशन कोड बदलने की आवश्यकता नहीं है। यह एक सर्विस और नेटवर्क के बीच बुनियादी ढांचे की एक परत है, जिसे सर्विस डिप्लॉयमेंट के साथ जोड़ा जाता है, जिसे आमतौर पर सर्विस मैश के रूप में जाना जाता है। इस्तियो का कंट्रोल प्लेन अंतर्निहित क्लस्टर प्रबंधन प्लैटफ़ॉर्म को हटा देता है, जो कुबेरनेट्स, मेसोस्फीयर आदि हो सकता है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/istio.md:17]:
```suggestion
Istio को जोड़ने के लिए एप्लिकेशन कोड बदलने की आवश्यकता नहीं है। यह एक सर्विस और नेटवर्क के बीच बुनियादी ढांचे की एक परत है, जिसे जब सर्विस डिप्लॉयमेंट के साथ जोड़ा जाता है, तो इसे आमतौर पर सर्विस मैश (Service Mesh) के रूप में भी जाना जाता है। Istio का कंट्रोल प्लेन अंतर्निहित क्लस्टर प्रबंधन प्लैटफ़ॉर्म को अलग कर देता है, जो कुबेरनेट्स, Mesosphere आदि हो सकते हैं।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from myside, hope you find them useful. Thanks

**Review by Babapool — COMMENTED** [review]
The changes look okay to me. (I am not an official `hi` localization approver/reviewer). Thanks.
cc: @Garima-Negi @divya-mohan0209

**Review by divya-mohan0209 — COMMENTED** [review]
Thank you for this PR.
/approve

**Pull request merged**